### ISSUE TITLE
Docker workflow broken due to "manifest list" problem

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#3902]
## Description

Our docker builds for all branches have started failing:

https://github.com/sigp/lighthouse/actions/runs/3965393971

Seems to be due to some tooling update on Docker's side, the type-of-thing that we previously expected (a "manifest") is now something else (a "manifest list"). Not sure what the fix is.

**Comment by antondlr** [comment]
the issue appears to be that the images we're amending to the manifest list is itself a manifest list with the following properties:
```
Image: docker.io/sigp/lighthouse:latest-arm64-unstable-modern (digest: sha256:657908728a564471c7d2eeca0ccde7d0ac3281864a79ac4d53ce314b398a4965)
 * Manifest List: Yes (Image type: application/vnd.oci.image.index.v1+json)
 * Supported platforms:
   - linux/arm64
   - unknown/unknown
```
this isnt the case for our latest release:

```
Image: docker.io/sigp/lighthouse:v2.3.0-arm64-modern (digest: sha256:1b3feb1ed864effa584fed632504a8fd3477c9bc7abd6d6e4b964be2f330cbe8)
 * Manifest List: No (Image type: application/vnd.docker.distribution.manifest.v2+json)
 * Supports: linux/arm64
 ```
 
 so far I've been unable to reproduce (=build + push image with `unknown/unknown` platform added) when mirroring the CI's build params 
 
 tool to inspect:
 `docker run --rm mplatform/mquery sigp/lighthouse:latest-arm64-unstable` 
 (image needs to be pushed to docker hub, doesn't work for local images)

**Comment by michaelsproul** [comment]
Closed by #3904!
